pkg/scaffold/util/util.go

```diff
@@ -26,7 +26,8 @@ import (
 	"sigs.k8s.io/kubebuilder/pkg/scaffold/resource"
 )
 
-func GetResourceInfo(r *resource.Resource, repo, domain string, isMultiGroup bool) (resourcePackage, groupDomain string) {
+func GetResourceInfo(r *resource.Resource, repo, domain string,
```

Comment:
This isn't that long of a signature. Can you add a nolint here? More generally I think long method names are fine (to a point).

Reply:
It is bigger than the standard defined. 
Will required to scroll.